src/dagmc_bridge.h

```diff
@@ -3,13 +3,17 @@
 
 /* Types.hpp may be validly included from C code */
 #include <moab/Types.hpp>
+#include <DagMC.hpp>
 
 #ifdef __cplusplus
 using moab::ErrorCode;
 using moab::EntityHandle;
+using moab::DagMC;
 
 namespace pyne {
 
+static DagMC* DAG = new DagMC();
```

Comment:
We may want to check this for memory leaks... I don't see a `delete` keyword anywhere in `dagmc_bridge.*`

Reply:
Whoops, there are a couple but not for the `DAG` pointer.

Reply:
I think the fact that this is static means we should be ok